src/main/java/com/hivemq/cli/commands/hivemq/export/clients/ExportClientsCommand.java

```diff
@@ -182,14 +182,18 @@ public ExportCompletedHandler(final @NotNull ClientDetailsCsvWriterTask clientDe
         public @NotNull Integer apply(Void o, Throwable throwable) {
             printingScheduler.shutdown();
             if (throwable != null) {
-                System.err.println("\rFailed to retrieve client details: " + Throwables.getRootCause(throwable).getMessage());
-                if (throwable.getCause() instanceof ApiException) {
-                    final ApiException apiException = (ApiException) throwable.getCause();
-                    if (apiException.getResponseBody() != null) {
-                        final Gson gson = new GsonBuilder().setPrettyPrinting().create();
-                        final JsonElement je = JsonParser.parseString(apiException.getResponseBody());
-                        final String jsonString = gson.toJson(je);
-                        System.err.println(jsonString);
+                if (throwable.getCause() instanceof JsonParseException) {
+                    System.err.println("\rFailed to retrieve client details. Please check the URL for the HiveMQ REST-API");
+                } else {
+                    System.err.println("\rFailed to retrieve client details: " + Throwables.getRootCause(throwable).getMessage());
+                    if (throwable.getCause() instanceof ApiException) {
+                        final ApiException apiException = (ApiException) throwable.getCause();
+                        if (apiException.getResponseBody() != null) {
+                            final Gson gson = new GsonBuilder().setPrettyPrinting().create();
+                            final JsonElement je = JsonParser.parseString(apiException.getResponseBody());
+                            final String jsonString = gson.toJson(je);
+                            System.err.println(jsonString);
+                        }
                     }
```

Comment:
I think this code is unreachable because the JsonParseException can only be thrown in line `193`. 

1. The CLI will try to contact an url with a rest api
2. The rest client of the cli throws an exception in one of the tasks which will be wrapped as a `CompletionException`, e. g. contacting `https://google.com` - throws a `CompletionException` with a `ApiException` as cause exception.
3.  Because google answers the call with an html response body instead a json as the hivemq api would, the parsing will fail in line `193`

Therefore, I think the code you are looking for is the following:
```
if (throwable.getCause() instanceof ApiException) {
    final ApiException apiException = (ApiException) throwable.getCause();
    if (apiException.getResponseBody() != null) {
        final Gson gson = new GsonBuilder().setPrettyPrinting().create();
        try {
            final JsonElement je = JsonParser.parseString(apiException.getResponseBody());
            final String jsonString = gson.toJson(je);
            System.err.println("\rFailed to retrieve client details: " + Throwables.getRootCause(throwable).getMessage());
            System.err.println(jsonString);
        } catch (final JsonSyntaxException jsonEx) {
            System.err.println("\rFailed to retrieve client details. Please check the URL for the HiveMQ REST-API");
        }

    }
}
else {
    System.err.println("\rFailed to retrieve client details: " + Throwables.getRootCause(throwable).getMessage());
}
```